packages/get-platform/package.json

```diff
@@ -36,6 +33,9 @@
     "terminal-link": "2.1.1",
     "ts-pattern": "5.0.5"
   },
+  "dependencies": {
+    "execa": "5.1.1"
+  },
```

Comment:
Just curious, is there something special going on with execa?

Reply:
The type bundler was not able to bundle execa types, crashes with an error.